### ISSUE TITLE
Written Instructions

### ISSUE THREAD

**Issue by RezaAmbler** [librespeed/speedtest#495]
Could you put together written instructions. It would be much more preferable to a video walk through.
Thanks

**Comment by leswaters** [comment]
Yes, I second this request.  The video wanders all over the place, installing PHP, etc. etc.  I just want a step by step of what to install. No rpm. Just what files where.  Thank you.
